ceph.spec.in

```diff
@@ -537,6 +537,7 @@ Requires:       python%{python3_pkgversion}-cherrypy
 Requires:       python%{python3_pkgversion}-jwt
 Requires:       python%{python3_pkgversion}-routes
 Requires:       python%{python3_pkgversion}-werkzeug
+Suggests:       python%{python3_pkgversion}-saml
 %endif
 %if 0%{?suse_version}
```

Comment:
If this is going to be added, can we do the same for SUSE, where the package name is `python3-python3-saml` (yes, confirmed, that is really the name)

i.e.

```
Recommends:    python%{python3_pkgversion}-python3-saml
```

here in this `suse_version` if block

Reply:
see also https://pkgs.org/download/python3-python3-saml

Reply:
Thanks @smithfarm! Done.